app/services/api/charts/company.rb

```diff
@@ -0,0 +1,110 @@
+module Api
+  module Charts
+    class Company
+      SECTORS_LEVELS_DESC = {
+        '0' => 'Unaware of Climate Change as a Business Issue',
+        '1' => 'Acknowledging Climate Change as a Business Issue',
+        '2' => 'Building Capacity',
+        '3' => 'Integrating into Operational Decision Making',
+        '4' => 'Strategic Assessment'
+      }.freeze
+
+      def details_data(company)
```

Comment:
Since you use `company` in almost all method, you can move `company` as an instance variable and assign it in initialize method

Reply:
fixed in fbefce8